bluemira/codes/wrapper.py

```diff
@@ -24,6 +24,7 @@ def systems_code_solver(
     params: ParameterFrame,
     build_config: BuildConfig,
     module: str = "PROCESS",
+    params_cls=None,
```

Comment:
you'll also need to put the `MappedParameterFrame` import in the TYPE_CHECKING block
```suggestion
    params_cls: type[MappedParameterFrame] | None =None,
```